nano/store/write_queue.hpp

```diff
@@ -38,27 +38,27 @@ class write_guard final
  * Allocates database write access in a fair maner rather than directly waiting for mutex aquisition
  * Users should wait() for access to database write transaction and hold the write_guard until complete
  */
-class write_database_queue final
+class write_queue final
 {
 public:
-	write_database_queue (bool use_noops_a);
+	write_queue (bool use_noops_a);
```

Comment:
Since the class is getting a small refresh, adding explicit here will silence warnings about implicit conversions.